libs/datasets/timeseries.py

```diff
@@ -175,6 +175,16 @@ def get_subset(
         )
         return self.__class__(self.data.loc[row_binary_array, :])
 
+    def get_columns_and_date_subset(
```

Comment:
Thoughts on just adding these additional args to `get_subset`? They seem useful

Reply:
I thought about it but decided to not overload get_subset more and keep the existing flow as much as possible until I have more confidence that I can quickly work out that i've broken something.